cdap-security/src/test/java/co/cask/cdap/security/securestore/FileSecureStoreTest.java

```diff
@@ -117,12 +118,13 @@ public void testGetMetadata() throws IOException {
   }
 
   @Test
-  public void testOverwrite() throws IOException {
+  public void testOverwrite() throws IOException, InterruptedException {
     secureStoreManager.put(KEY1, VALUE1.getBytes(Charsets.UTF_8), PROPERTIES_1);
     SecureStoreData oldData = secureStore.get(KEY1);
     long oldCreateTime = oldData.getMetadata().getLastModifiedTime();
     Assert.assertArrayEquals(VALUE1.getBytes(Charsets.UTF_8), oldData.get());
     String newVal = "New value";
+    Thread.sleep(1000);
```

Comment:
The test was a little flaky without that, sometimes both the puts happened in the same ms. I ran it a  bunch of times now without the sleep and it is consistently passing. So, I am removing the sleep.